optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-backend/src/test/java/org/optaplanner/workbench/screens/domaineditor/backend/server/helper/PlanningSolutionCopyHelperTest.java

```diff
@@ -66,9 +66,8 @@ public void setUp() {
         copyHelper = new PlanningSolutionCopyHelper(ioService,
                                                     dataModelerService,
                                                     kieModuleService);
-        when(ioService.readAllString(Paths.convert(sourcePath))).thenReturn("test source");
         when(dataModelerService.loadDataObject(any(),
-                                               anyString(),
+                                               any(),
```

Comment:
What is the reason for this type of changes? It seems unnecessary.

Reply:
Mockito 3 now differentiates between `anyString()`, which does not accept null values, and `any()`, which does.